fedimint-server/src/config/mod.rs

```diff
@@ -241,7 +241,7 @@ impl ServerConfig {
             modules: Default::default(),
         };
         let consensus = ServerConfigConsensus {
-            code_version: CODE_VERSION.to_string(),
+            code_version: version_hash,
```

Comment:
**note:** Importantly before two different custom versions of Fedimint would not fail the version check during dkg.